src/GRA.Web/Areas/MissionControl/Views/Roles/Detail.cshtml

```diff
@@ -31,7 +42,7 @@
                 </div>
 
                 <div class="col-xs-6">
-                    <h3>Has Permissions</h3>
+                    <h3>Has Permissions (@Model.SelectedPermissions?.Count())</h3>
```

Comment:
This should be an `h2` with the `class="h3"`.  With those changes the view looks the same. This is just an accessibility issue we should get in routine of.

Reply:
Good catches on these, I'll try to be more on top of this!